### ISSUE TITLE
[SEARCH] Error on highlight query with empty value

### ISSUE THREAD

**Issue by mpomet** [orientechnologies/orientdb#9134]
## Expected behavior  
Using highlight in SEARCH_CLASS should work even if a value is null.

## Actual behavior  
Using highlight in SEARCH_CLASS return an error if an indexed field has null/empty value.      
`java.lang.IllegalArgumentException: Field title in document is not stored and cannot be analyzed`

## Steps to reproduce  
```SQL
CREATE CLASS MyClass EXTENDS V;
CREATE PROPERTY MyClass.title STRING;
CREATE PROPERTY MyClass.description STRING;

CREATE INDEX MyClass.fulltxtindex ON MyClass(title, description) FULLTEXT ENGINE LUCENE;

INSERT INTO MyClass (description) VALUES ("Test 1 !");

SELECT $title_hl, $description_hl FROM MyClass
WHERE SEARCH_CLASS("test", {
    "highlight": {
        "fields": ["title", "description"]
    }
}) = true

```

**Comment by luigidellaquila** [comment]
@wolf4ood any idea?

**Comment by wolf4ood** [comment]
Hi @luigidellaquila @mpomet 

let me check

**Comment by wolf4ood** [comment]
Hi @mpomet 

i've just  pushed a fix. It will be available in the next 3.0.29 GA release.

Thanks
